src/components/CollectionsPage/OptionsDropdown/OptionsDropdown.module.css

```diff
@@ -0,0 +1,24 @@
+.OptionsDropdown:global(.ui.dropdown) {
+  position: absolute;
+  top: 5px;
+  right: 10px;
+  z-index: 10000000;
+  display: inline-block;
+}
+
+.OptionsDropdown:global(.ui.dropdown) > :global(.dropdown.icon):before {
+  content: '…' !important;
+  color: black !important;
```

Comment:
I think these `!important` are not required, right?

Also this is 100% subjective but I think the black dots are a bit jarring to see, not sure if white's better, but maybe we can try using one of the grays we use on the rest of the site?

Reply:
Removed the important and re-colored the dots.